consumers/campaigner/main.go

```diff
@@ -67,127 +91,121 @@ func (h *MessageHandler) HandleMessage(m *nsq.Message) error {
 				"user_id":     msg.UserID,
 				"segment_ids": msg.SegmentIDs,
 			}).WithError(err).Error("Unable to fetch subscribers.")
```

Comment:
Stay consistent with the syntax and wording in all log messages:

```suggestion
			}).WithError(err).Error("unable to fetch subscribers")
```